galaxy/importer/linters/__init__.py

```diff
@@ -72,3 +72,29 @@ def _check_files(self, paths):
         for line in proc.stdout:
             yield line.strip()
         proc.wait()
+
+
+class AnsibleLinter(BaseLinter):
+
+    cmd = 'ansible-lint'
+
+    def _check_files(self, paths):
+        rules_folder = '/galaxy/galaxy/importer/galaxy-lint-rules/rules'
```

Comment:
1. Hardcoded docker specific path.
2. Directory term is preferred to use.

Reply:
3. In general, I'd advise using pathlib as an underlying implementation of working with paths. (See pathlib2 for a good and improved backport from stdlib)

Reply:
I started to use os.path to build a relative path, but then re-added hardcoded path since I moved repo location to root.